packages/workbox-precaching/_default.mjs

```diff
@@ -76,44 +76,53 @@ const _removeIgnoreUrlParams = (origUrlObject, ignoreUrlParametersMatching) => {
 const _getPrecachedUrl = (url, {
   ignoreUrlParametersMatching = [/^utm_/],
   directoryIndex = 'index.html',
+  cleanUrls = true,
+  urlManipulation = null,
 } = {}) => {
   const urlObject = new URL(url, location);
 
-  // If we precache '/some-url' but the URL referenced from the browser
-  // is '/some-url#1234', the comparison won't work unless we normalise
-  // the URLS.
-  // See https://github.com/GoogleChrome/workbox/issues/488.
+  // Change '/some-url#123' => '/some-url'
   urlObject.hash = '';
 
-  const cachedUrls = precacheController.getCachedUrls();
-  if (cachedUrls.indexOf(urlObject.href) !== -1) {
-    // It's a perfect match
-    if (process.env.NODE_ENV !== 'production') {
-      logger.debug(`Precaching found an exact URL match for ` +
-        getFriendlyURL(urlObject.toString()));
-    }
-    return urlObject.href;
-  }
-
-  let strippedUrl = _removeIgnoreUrlParams(
+  const noSearchUrl = _removeIgnoreUrlParams(
```

Comment:
`noSearchUrl` sounds like it's completely dropping the `search` portion, which isn't the case. We're just removing the ignored search parameters. Could you name it `urlWithoutIgnoredParams` or something similar?